src/olympia/lib/settings_base.py

```diff
@@ -1101,8 +1101,8 @@ def lazy_langs(languages):
 # using the JSON format but still accept pickled messages.
 # TODO: Remove `pickle` once the upgrade is done and seems stable.
 CELERY_ACCEPT_CONTENT = ['pickle', 'json']
-CELERY_TASK_SERIALIZER = 'json'
```

Comment:
The comment above doesn't make sense anymore with those changes.

Reply:
Fixed in 41ac7cc5da